content/docs/user-guide/troubleshooting.md

```diff
@@ -76,6 +76,12 @@ valid targets for `dvc add`. If the target path is a directory symlink, or if
 the target path contains any intermediate directory symlinks, `dvc add` will
 fail.
 
+## Cannot import multiple files or directories at once {#import-wildcard}
+
+DVC only supports a single file or directory as valid  target for `dvc import`.
+If the target path contains a wildcard that matches multiple files or directories,
+ `dvc import` will fail.
```

Comment:
Thanks for adding this 🙂 

I left a question about it in https://github.com/iterative/dvc/pull/5010#pullrequestreview-544289052 though.